cmd/antrea-agent/options.go

```diff
@@ -306,6 +306,20 @@ func (o *Options) validateAntreaIPAMConfig() error {
 	return nil
 }
 
+func (o *Options) validateMulticlusterConfig(encapMode config.TrafficEncapModeType) error {
+	if (features.DefaultFeatureGate.Enabled(features.Multicluster) || o.config.Multicluster.Enable) &&
```

Comment:
First check `!features.DefaultFeatureGate.Enabled(features.Multicluster) && o.config.Multicluster.Enable`. Maybe we can do the following:
```
if !o.config.Multicluster.Enable {    
    return nil
}

if !features.DefaultFeatureGate.Enabled(features.Multicluster) {
    return error
}
```

Reply:
Thanks. Changed.